Readme.md

```diff
@@ -1,16 +1,16 @@
-# Lombiq <add project name here>
+# Lombiq Hosting Tenants
 
 
 
 ## About
 
-Add a general overview of the project here. Don't forget to update the year in the Licence!
+Various libraries that help to build a multi-tenant web application in Orchard Core. 
 
 
 ## Documentation
 
-Add detailed documentation here. If it's a lot of content then create documentation pages under the *Docs* folder and link pages here.
-
+The module consists of the following independent libraries (all in their own features):
+- [Lombiq Hosting Tenants - Admin](Lombiq.Hosting.Tenants.Admin/Readme.md)
 
 ## Contributing and support
```

Comment:
Formázási szabályokért lásd: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/12156958/Adding+documentation+pages+to+our+projects Továbbá: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/10453016/Documenting+our+projects

```suggestion


## Contributing and support
```